src/main/java/uk/gov/hmcts/darts/task/service/impl/AutomatedTaskServiceImpl.java

```diff
@@ -56,8 +57,15 @@ public class AutomatedTaskServiceImpl implements AutomatedTaskService {
     private final UserIdentity userIdentity;
     private final List<AutoloadingAutomatedTask> autoloadingAutomatedTasks;
 
+    @Value("${darts.automated.allow-automated-tasks:true}")
+    private final boolean allowAutomatedTasks;
+
     @Override
     public void configureAndLoadAutomatedTasks(ScheduledTaskRegistrar taskRegistrar) {
+        if (!allowAutomatedTasks) {
+            log.info("Automated tasks are disabled");
+            return;
+        }
```

Comment:
Is this change intentional? It doesn't seem relevant for the ticket.

Reply:
I spoke to Jack on this one and he is happy as it makes local testing easier if these are disabled (It will be enabled on all deployed envs by default)